game/src/challenges.rs

```diff
@@ -284,7 +284,7 @@ impl State for ChallengesPicker {
     }
 
     fn draw(&self, g: &mut GfxCtx, app: &App) {
-        g.clear(app.cs.grass);
+        g.clear(app.cs.dialog_bg);
```

Comment:
We were using "grass" to color lots of things green which weren't actually grass. I've tried to identify the non-grass places we were using the green color and called it "dialog_bg". LMK if you have a better name in mind.

For example, the background of the main screen is "grass" colored, but presumably we don't want it to be "grass" textured.
<img width="1792" alt="Screen Shot 2020-09-18 at 12 32 11 AM" src="https://user-images.githubusercontent.com/217057/93570144-6b839080-f947-11ea-8450-a46a9ee02594.png">

Reply:
The provenance of using `grass` for the UI: at one of the first democracylab hackathons, a few designers jumped on the project and made this initial pre-game screen; previously it was just a menu on top of the scrolling splash screen. They liked the green grass, which was just pawned from OSM's color scheme.